packages/orchestration/src/chain-info.js

```diff
@@ -10,7 +10,9 @@ import fetchedChainInfo from './fetched-chain-info.js';
 const knownChains = /** @satisfies {Record<string, ChainInfo>} */ (
   harden({
     ...fetchedChainInfo,
-    agoric: {
+    // XXX does not have useful connections
+    // UNTIL https://github.com/Agoric/agoric-sdk/issues/9492
+    agoriclocal: {
```

Comment:
using `agoriclocal` as both a chain id and a chain name made me scratch my head a bit. `aglocal`? not sure that helps much. The XXX suffices, I suppose.